databaseAnnotator/src/main/java/org/cbioportal/database/DatabaseAnnotator.java

```diff
@@ -50,38 +50,35 @@
 @SpringBootApplication
 public class DatabaseAnnotator {
 
-    private static Logger log = Logger.getLogger(DatabaseAnnotator.class);
+    private static Logger LOG = Logger.getLogger(DatabaseAnnotator.class);
```

Comment:
In some places we're using `LOG`, in others `log`. It seems like we're using `LOG` more often